openApiDocs/v1.0/Identity.AccessReview.yml

```diff
@@ -0,0 +1,19 @@
+openapi: 3.0.1
+info:
+  title: Identity.AccessReview
+  version: v1.0
+servers:
+  - url: https://graph.microsoft.com/v1.0/
+    description: Core
+paths: { }
```

Comment:
We should not download documents with empty paths. DevX API should return a 400 bad request.

Reply:
https://github.com/microsoftgraph/microsoft-graph-explorer-api/issues/246

Reply:
Checking the Download script, and DevX api code, when the download script encounters a 400 it doesn't create a file.

Reply:
Seems this is not an issue with the download script.